app/views/search/results/partials/_project_text.html.erb

```diff
@@ -4,15 +4,15 @@
 
 <% if project.archived? %>
   <span class="label label-warning"><%=t 'search.index.archived' %></span>
-  <% if can_read_team?(project.team) and can_restore_project(project) %>
+  <% if can_read_team?(project.team) and can_restore_project?(project) %>
```

Comment:
Coul we also get rid  of `can_read_team?(project.team)`, or does this "exception" make sense?

Reply:
We can probably throw it out. This "duo" has been here since 2 years ago. And I don't think there's a case where you cannot read the team, but can restore the project.